connector/oidc/oidc_test.go

```diff
@@ -92,6 +93,25 @@ func TestHandleCallback(t *testing.T) {
 				"email_verified": true,
 			},
 		},
+		{
+			name:                 "overrideWithCustomEmailClaim",
+			userIDKey:            "", // not configured
+			userNameKey:          "", // not configured
+			overrideClaimMapping: true,
+			claimMapping: ClaimMapping{
+				EmailKey: "custommail",
+			},
+			expectUserID:       "subvalue",
+			expectUserName:     "namevalue",
+			expectedEmailField: "customemailvalue",
+			token: map[string]interface{}{
+				"sub":            "subvalue",
+				"name":           "namevalue",
+				"mail":           "emailvalue",
```

Comment:
```suggestion
				"email":           "emailvalue",
```

I think this should be `email` (potential copy-paste error from the above custom claim example?)

Reply:
Done